hawkbit-ui/src/main/java/org/eclipse/hawkbit/ui/artifacts/upload/AbstractFileTransferHandler.java

```diff
@@ -194,11 +203,14 @@ protected void publishUploadFailedAndFinishedEvent(final FileUploadId fileUpload
         LOG.info("Upload failed for file {} due to reason: {}, exception: {}", fileUploadId, failureReason,
```

Comment:
Check if logging the message could be done when catching the exception

Reply:
also **consider** to change other log messages here (failed/succeeded) to debug

Reply:
Adjusted logging levels to debug for client errors, only kept *start* and *failed*/*success* messages as info; moved log message.